packages/pyxis-react/src/stories/components/DateInput/Overview.tsx

```diff
@@ -116,7 +153,7 @@ const Overview: FC = () => (
   <>
     <OverviewTemplate category="Component" description={overviewDescription} isMain title="DateInput">
       <Canvas>
-        <Story id="components-form-dateinput-all-stories--default" />
+        <Story id="components-form-DateInput-all-stories--inside-form-item" />
```

Comment:
```suggestion
        <Story id="components-form-dateinput-all-stories--default" />
```
La storia "inside-form-item" non esiste più